govcd/vm_test.go

```diff
@@ -967,15 +967,34 @@ func (vcd *TestVCD) Test_VMSetGetGuestCustomizationSection(check *C) {
 	guestCustomizationPropertyTester(vcd, check, vm)
 }
 
-// Test gathering VM virtual hardware items
-func (vcd *TestVCD) Test_GetVirtualHardwareSection(check *C) {
-	itemName := "TestGetVirtualHardwareSection"
+// Test create internal disk For VM
+func (vcd *TestVCD) Test_AddInternalDisk(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+
+	vm, storageProfile, diskSettings, diskId, err := vcd.createInternalDisk(check, 1)
+	check.Assert(err, IsNil)
+
+	//verify
+	disk, err := vm.GetInternalDiskById(diskId, true)
+	check.Assert(err, IsNil)
 
+	check.Assert(disk.StorageProfile.HREF, Equals, storageProfile.HREF)
```

Comment:
Please add a check for ID correspondence

Reply:
added